lib/pbench/server/api/resources/query_apis/iteration_samples.py

```diff
@@ -67,20 +66,19 @@ def preprocess(self, client_json: JSON) -> CONTEXT:
         return {"dataset": dataset, "run_id": run_id}
 
 
-class IterationSampleRows(IterationSamples):
+class IterationSampleNamespace(IterationSamples):
     """
-    Iteration samples API that returns only unique rows for each subdocument.
+    Iteration samples API that returns namespaces for each whitelisted
+    result-data-sample subdocuments.
```

Comment:
I think we're thinking that the "namespace" contains the list of what the Dashboard is going to use a column headers and filter keys (rather than thinking of each key as a namespace for which we are returning the available values).  So, we're returning the names from the "space" as well as lists of available values for each name.